src/finfield.c

```diff
@@ -542,10 +543,10 @@ FF              FiniteField (
       else
```

Comment:
I don't care either way, but I am not going to add them in this PR, and also wanted to "blend in" with the existing code. It's hard enough already to stop myself from reformatting every function and file I touch ;-)